app/src/main/java/com/github/steroidteam/todolist/todo/Task.java

```diff
@@ -45,36 +44,18 @@ public String getBody() {
         return new String(body);
```

Comment:
Just nitpicking but I think new String() is necessary if String is immutable

Reply:
Fixed, thanks!